### ISSUE TITLE
Feature Request: Arbitrary Metadata Editor

### ISSUE THREAD

**Issue by tpwrules** [CloudCompare/CloudCompare#1868]
**Describe the feature you would like**
To be able to edit the items listed under "Meta data" in a cloud's properties.

**Describe alternatives you've considered**
Metadata seems to be associated with certain types of files, so maybe it was never meant to be edited. Some properties can be edited with certain types of exporters but not everything.

**Is your feature request related to a problem? Please describe.**
For example, I would like to be able to edit the CRS in a LAS file.

**Comment by dgirardeau** [comment]
Yes, meta data are really there for some plugins to store specific/custom data between two calls of the plugin (typically when loading a LAS file, and saving it later, and the plugin needs to remember some LAS-specific entries).

We don't really want the user to mess with it... Or it would have to be done in a kind of 'advanced' mode.
